### PR TITLE
Column level lineage and path test

### PR DESCRIPTION
**Pull request by kkorchak**
Test steps:
1. Open Dataset with column-level lineage configured
2. Navigate to Lineage tab -> Visualize Lineage
3. Enable “Show Columns” switch
4. Verify the functionality of column lineage
5. Open dataset Impact Analysis view, enable Column Lineage
6. Verify Upstream and Downstream column lineage
7. Verify Column Path modal works as expected

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: smoke-test/tests/cypress/cypress/e2e/lineage/lineage_column_path.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      cy.clickOptionWithText("Upstream");
+      cy.waitTextVisible("SampleCypressKafkaDataset");
+      cy.ensureTextNotPresent("field_bar");
+      cy.contains("Select column").click({ force: true}).wait(1000);
+      cy.get(".rc-virtual-list").contains("shipment_info").click(); 
+      cy.waitTextVisible("field_bar");
+      cy.clickOptionWithText("field_bar");
+      cy.get('[data-testid="entity-paths-modal"]').contains("Column path").should("be.visible");
+      cy.get('[data-testid="entity-paths-modal"]').contains("field_bar").should("be.visible");
+      cy.get('[data-testid="entity-paths-modal"]').contains("shipment_info").should("be.visible");
```
Comment (jjoyce0510) [inline smoke-test/tests/cypress/cypress/e2e/lineage/lineage_column_path.js:51]: any way to simplify?

Comment (kkorchak) [reply smoke-test/tests/cypress/cypress/e2e/lineage/lineage_column_path.js]: done

File: smoke-test/tests/cypress/cypress/e2e/lineage/lineage_column_path.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import { aliasQuery } from "../utils";
+const DATASET_ENTITY_TYPE = 'dataset';
+const DATASET_URN = 'urn:li:dataset:(urn:li:dataPlatform:hdfs,SampleCypressHdfsDataset,PROD)';
+const DOWNSTREAM_DATASET_URN = "urn:li:dataset:(urn:li:dataPlatform:kafka,SampleCypressKafkaDataset,PROD)";
+const upstream = '[data-testid="node-urn:li:dataset:(urn:li:dataPlatform:kafka,SampleCypressKafkaDataset,PROD)-Upstream"] text';
```
Comment (jjoyce0510) [inline smoke-test/tests/cypress/cypress/e2e/lineage/lineage_column_path.js:5]: if this is clicking on upstream column, name the variable upstreamColumn

so it's clear that the mouse click is on the column level box, and not the overall parent node

**Review by jjoyce0510 — APPROVED** [review]
LGTM

File: smoke-test/tests/cypress/cypress/e2e/lineage/lineage_column_path.js (hunk 2 above)
Comment (kkorchak) [reply smoke-test/tests/cypress/cypress/e2e/lineage/lineage_column_path.js]: done

**Pull request merged**